src/lib/i18n/use-safe-translations.ts

```diff
@@ -0,0 +1,15 @@
+import { useTranslations } from "next-intl"
+export const useSafeTranslations = (ns?: string) => {
+  const _t = useTranslations(ns)
+  return (key: string | undefined, props: Record<string, any> = {}) => {
+    if (key) {
+      const t = _t(key, { ...props })
+      if (t) {
+        return t
+      }
+    }
+    return process.env.NEXT_PUBLIC_TRANSLATION_HELPERS
+      ? `Missing translation ${ns}:${key}`
+      : ""
+  }
```

Comment:
Why did you wrap `useTranslations`? This breaks features like the type-safety, I'd really recommend not doing this.